aws-datastore/src/androidTest/java/com/amplifyframework/datastore/storage/sqlite/SQLiteStorageAdapterModelConverterTest.java

```diff
@@ -70,7 +71,7 @@ public void saveModelWithAllTypesThenQuery() throws DataStoreException {
         final Todo todo = createTestTodoModel();
         this.adapter.save(todo);
 
-        final List<Todo> result = this.adapter.query(Todo.class, Todo.ID.eq(todo.getId()));
+        final List<Todo> result = this.adapter.query(Todo.class, byId(todo.getId()));
```

Comment:
I like the concept of the `Where` as a class to avoid the static imports (even though I'm torn about the reasons why the Java community perceives static imports a something generally bad).

I updated the readme with more details and will push an update to this PR introducing this way to build the DSL.